Add support for artifacts in TMT/FMF plans

phracek opened the pull request:
This pull request adds support for artifacts in TMT/FMF plans

Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

phracek commented:
https://github.com/sclorg/testing-farm-as-github-action/issues/16

fernflower commented on README.md:
```diff
@@ -17,19 +17,20 @@ API key to the Testing Farm should be stored in your organization's secrets to s
+| `pull_request_status_name`   | GitHub pull request status name                                                               | Fedora                        |
+| `env_vars`                   | Environment variables for test env, separated by ;                                            | empty                         |
+| `env_secrets`                | Environment secrets for test env, separated by ;                                              | empty                         |
+| `env_artifacts`              | Environment artifacts for test env, separated by ;                                            | empty                         |
```
Maybe just `artifacts` or `test_artifacts` are a better fit here in terms of naming?

fernflower commented on action.yml:
```diff
@@ -47,6 +47,14 @@ inputs:
+    required: false
+    default: 'epel-7-x86_64'
+  env_artifacts:
+    description: 'Environment artifacts for test env, separated by ;'
```
As long as only `fedora-copr-build` artifacts are supported, maybe it should be mentioned in the input description?

phracek commented:
@fernflower I have addressed all issues. Feel free to comment / improve / approved.

As soon as this pull request is merged, I will release a new version.

fernflower reviewed (approved):
lgtm!
